shared/tests/unit/core/test_core_exceptions.py

```diff
@@ -0,0 +1,17 @@
+from shared.core import ModelDoesNotExist, ModelExists, ModelLocked, ModelNotDeleted, InvalidFormat
```

Comment:
Please check tavis config: The `InvalidFormat` import is not used. Somthing must have done an alert (flake8 i guess)...